lib/amazonka-core/src/Amazonka/Data/Path.hs

```diff
@@ -74,6 +79,13 @@ escapePath :: Path a -> EscapedPath
 escapePath (Raw xs) = Encoded (map (URI.urlEncode True) xs)
 escapePath (Encoded xs) = Encoded xs
 
+-- | Escape a path twice for rare occasions like computing the sigv4 canonical path.
```

Comment:
```suggestion
-- | Escape a path twice. Used when computing the SigV4 canonical path.
```
Reason: it's nearly every AWS request, so better not to call it rare.

Reply:
Probably better to put the "Used in SigV4" on the `newtype` declaration so when someone clicks `TwiceEscapedPath` in haddock, they see it.